osu.Framework/Configuration/IReadonlyBindableCollection.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu-framework/master/LICENCE
+
+using System;
+using System.Collections.Generic;
+
+namespace osu.Framework.Configuration
+{
+    public interface IReadonlyBindableCollection<out T> : IReadOnlyCollection<T>
```

Comment:
```suggestion
    public interface IReadOnlyBindableCollection<out T> : IReadOnlyCollection<T>
```

Will require fixes elsewhere, best to do in code rather than accepting suggestion.

Reply:
In-fact, along with my suggestion to remove `AddRange` from `IBindableCollection<T>`, I suggest combining this interface into `IBindableCollection<T>`.

Reply:
I agree to the rename.

Combining this into IBindableCollection depends on the conversation above.